inlong-agent/agent-core/src/main/java/org/apache/inlong/agent/core/HeartbeatManager.java

```diff
@@ -228,7 +228,7 @@ private HeartbeatMsg buildHeartbeatMsg() {
             heartbeatMsg.setInCharges(inCharges);
         }
         if (StringUtils.isNotBlank(nodeTag)) {
-            heartbeatMsg.setNodeTag(nodeTag);
+            heartbeatMsg.setNodeLabel(nodeTag);
```

Comment:
Can there be a clear meaning of the name?

tag and lab are too general

Reply:
Table name has  alreay  embodied its meaning